deploy/helm/sumologic/conf/logs/fluentd/logs.kubernetes.sumologic.filter.conf

```diff
@@ -8,4 +8,4 @@ exclude_pod_regex {{ .Values.sumologic.logs.container.excludePodRegex | quote }}
 exclude_container_regex {{ .Values.sumologic.logs.container.excludeContainerRegex | quote }}
 exclude_host_regex {{ .Values.sumologic.logs.container.excludeHostRegex | quote }}
 per_container_annotations_enabled {{ .Values.sumologic.logs.container.perContainerAnnotationsEnabled }}
-per_container_annotation_prefixes {{ join "," .Values.sumologic.logs.container.perContainerAnnotationPrefixes }}
+per_container_annotation_prefixes {{ join "," .Values.sumologic.logs.container.perContainerAnnotationPrefixes | default "[]" }}
```

Comment:
Indirectly. By default, we have an empty space here, so the output is a text block ending with: `per_container_annotation_prefixes `. Prettier wants to remove what it sees as a trailing space from it. Instead of excluding the file from formatting, I changed the default format - FluentD accepts lists both comma-delimited and json - and I think it's better this way.